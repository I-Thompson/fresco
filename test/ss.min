ss.search
plot ss-init.plot
show
q
min
migrad
end
plot ss-fit.plot
