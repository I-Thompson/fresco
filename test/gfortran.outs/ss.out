SFRESCO - FRES 3.4: Search Coupled Reaction Channels

 Please give search file name
 SFresco search file = ss.search                                         

 Fresco input file = p-cd.nin                                          
   and output file = p-cd.frout                                        
 Search on  4 variables for   1 datasets,
  MINUIT RELEASE 96.03  INITIALIZED.   DIMENSIONS 100/ 50  EPSMAC=  0.89E-15

  Define SEARCH VARIABLES:

   Variable  1=r0         is potential KP= 1, line= 2 col= 2
     value   1.1710           step  0.0100, min,max   0.0000  0.0000

   Variable  2=V          is potential KP= 1, line= 2 col= 1
     value  52.5000           step  0.1000, min,max   0.0000  0.0000

   Variable  3=W          is potential KP= 1, line= 2 col= 4
     value   3.5000           step  0.1000, min,max   0.0000  0.0000

   Variable  4=WD         is potential KP= 1, line= 3 col= 4
     value   8.5000           step  0.1000, min,max   0.0000  0.0000

  Read definition of data set            1
  Read DATA SET  1 from file =                                                 
    of type  0           with absolute errors, CM  for state/excit  1  1,  idir= 1

  Angular cross section T00 for energy  27.900 MeV
  Read data set            1
            28  data points for lab energy    27.8999996    
  Angle CM  Datum     Absolute error
   22.000  0.5480      0.4400E-01
   26.000  0.4750      0.2400E-01
   30.000  0.4810      0.1400E-01
   34.000  0.5020      0.1000E-01
   38.000  0.4470      0.9000E-02
   42.000  0.2560      0.5000E-02
   46.000  0.1150      0.3000E-02
   50.000  0.1440      0.4000E-02
   54.000  0.3710      0.1100E-01
   58.000  0.6120      0.1200E-01
   62.000  0.6780      0.1400E-01
   66.000  0.4990      0.1000E-01
   70.000  0.2480      0.5000E-02
   74.000  0.8600E-01  0.3000E-02
   78.000  0.1190      0.4000E-02
   82.000  0.2890      0.9000E-02
   86.000  0.4630      0.1400E-01
   90.000  0.4850      0.1500E-01
   94.000  0.3980      0.1200E-01
   98.000  0.2350      0.7000E-02
  102.000  0.1230      0.4000E-02
  106.000  0.8700E-01  0.3000E-02
  110.000  0.1350      0.4000E-02
  114.000  0.1980      0.6000E-02
  118.000  0.2250      0.9000E-02
  122.000  0.2230      0.9000E-02
  126.000  0.1890      0.6000E-02
  130.000  0.1610      0.5000E-02

  FRESCO output to p-cd.frout*

 Calculate scattering at energies
   27.900


  Total ChiSq/N =    20.5232 from     20.523

 PARAMETER DEFINITIONS:
    NO.   NAME         VALUE      STEP SIZE      LIMITS
     1 'r0        '    1.1710      0.10000E-01     no limits
     2 'V         '    52.500      0.10000         no limits
     3 'W         '    3.5000      0.10000         no limits
     4 'WD        '    8.5000      0.10000         no limits
sfresco> 
  ChiSq/N =    20.523 from     20.523
  xmgr file written: ss-init.plot                                      
sfresco>     Var   1=r0         value     1.171000, step   0.0100, error   0.0000
    Var   2=V          value    52.500000, step   0.1000, error   0.0000
    Var   3=W          value     3.500000, step   0.1000, error   0.0000
    Var   4=WD         value     8.500000, step   0.1000, error   0.0000
sfresco> 
   Call MINUIT
   with NOGradient, STRategy=0, ERRordef=  0.0357
 **********
 **    1 **SET NOGRADIENT 
 **********
 **********
 **    2 **SET STRAT    0.000    
 **********

 NOW USING STRATEGY  0: MINIMIZE THE NUMBER OF CALLS TO FUNCTION

 **********
 **    3 **SET ERRORDEF  0.3571E-01
 **********
minuit>  **********
 **    4 **MIGRAD 
 **********

 FIRST CALL TO USER FUNCTION AT NEW START POINT, WITH IFLAG=4.
 START MIGRAD MINIMIZATION.  STRATEGY 0.  CONVERGENCE WHEN EDM .LT. 0.36E-05

 FCN=   20.52319     FROM MIGRAD    STATUS=INITIATE      10 CALLS       11 TOTAL
                     EDM= unknown      STRATEGY= 0      NO ERROR MATRIX       

  EXT PARAMETER               CURRENT GUESS       STEP         FIRST   
  NO.   NAME        VALUE          ERROR          SIZE      DERIVATIVE 
   1      r0        1.1710       0.10000E-01   0.10000E-01   -784.14    
   2      V         52.500       0.10000       0.10000       -8.1024    
   3      W         3.5000       0.10000       0.10000        18.764    
   4      WD        8.5000       0.10000       0.10000        9.1251    
                               ERR DEF= 0.357E-01
 MIGRAD FAILS TO FIND IMPROVEMENT
 MIGRAD FAILS WITH STRATEGY=0.   WILL TRY WITH STRATEGY=1.
 EIGENVALUES OF SECOND-DERIVATIVE MATRIX:
        -0.1991E+00  0.9947E+00  0.1003E+01  0.2202E+01
 MINUIT WARNING IN HESSE   
 ============== MATRIX FORCED POS-DEF BY ADDING  0.20129     TO DIAGONAL.

 FCN=   8.935225     FROM HESSE     STATUS=NOT POSDEF    31 CALLS      145 TOTAL
                     EDM=  0.89E-04    STRATEGY= 1      ERR MATRIX NOT POS-DEF

  EXT PARAMETER                APPROXIMATE        STEP         FIRST   
  NO.   NAME        VALUE          ERROR          SIZE      DERIVATIVE 
   1      r0        1.1814       0.25888E-03   0.21383E-05   -2.7858    
   2      V         52.184       0.47089E-01   0.22386E-03   0.73889E-01
   3      W         2.8955       0.64867       0.33305E-03  -0.26392E-02
   4      WD        8.1821        1.0332       0.41467E-03  -0.22654E-02
                               ERR DEF= 0.357E-01
 MIGRAD FAILS TO FIND IMPROVEMENT
 MIGRAD TERMINATED WITHOUT CONVERGENCE.

 FCN=   8.935225     FROM MIGRAD    STATUS=FAILED       149 CALLS      150 TOTAL
                     EDM=  0.89E-04    STRATEGY= 1      ERR MATRIX NOT POS-DEF

  EXT PARAMETER                APPROXIMATE        STEP         FIRST   
  NO.   NAME        VALUE          ERROR          SIZE      DERIVATIVE 
   1      r0        1.1814       0.25888E-03   -0.0000       -2.7858    
   2      V         52.184       0.47089E-01   -0.0000       0.73889E-01
   3      W         2.8955       0.64867       -0.0000      -0.26392E-02
   4      WD        8.1821        1.0332        0.0000      -0.22654E-02
                               ERR DEF= 0.357E-01

 EXTERNAL ERROR MATRIX.    NDIM=  50    NPAR=  4    ERR DEF= 0.357E-01
  0.670E-07 0.802E-05 0.129E-03-0.206E-03
  0.802E-05 0.222E-02 0.260E-01-0.415E-01
  0.129E-03 0.260E-01 0.421E+00-0.669E+00
 -0.206E-03-0.415E-01-0.669E+00 0.107E+01
 ERR MATRIX NOT POS-DEF

 PARAMETER  CORRELATION COEFFICIENTS
       NO.  GLOBAL     1     2     3     4
        1  0.77161  1.000 0.658 0.769-0.771
        2  0.85363  0.658 1.000 0.852-0.854
        3  0.99814  0.769 0.852 1.000-0.998
        4  0.99817 -0.771-0.854-0.998 1.000
 ERR MATRIX NOT POS-DEF
minuit>  **********
 **    5 **END 
 **********

 CALL TO USER FUNCTION WITH IFLAG = 3

 END OF MINUIT COMMAND INPUT.    RETURN TO USER PROGRAM.

sfresco> 
  ChiSq/N =     8.935 from      8.935
  xmgr file written: ss-fit.plot                                       
sfresco> 